content/help/faq/security/accessing-non-istio-services.md

```diff
@@ -23,7 +23,7 @@ EOF
 
 {{< tip >}}
 This destination rule is already added to the system as part of the
-[Istio installation with default mutual TLS](/docs/setup/kubernetes/quick-start/#option-2-install-istio-with-default-mutual-tls-authentication).
+[Istio installation with default mutual TLS](/docs/setup/kubernetes/install/kubernetes/#option-2-install-istio-with-default-mutual-tls-authentication).
```

Comment:
automatic mutual tls installation is deprecated in 1.1 and will be removed in 1.2.

Reply:
Beyond the scope of the PR.